packages/x-charts/src/ChartsTooltip/contentDisplayed.test.tsx

```diff
@@ -161,6 +167,51 @@ describe.skipIf(isJSDOM)('ChartsTooltip', () => {
         ]);
       });
     });
+
+    it('should render tooltip when using composing scatter series', async () => {
+      const series = [
+        {
+          type: 'scatter',
+          data: [
+            { x: 1, y: 10 },
+            { x: 2, y: 5 },
+          ],
+        },
+      ] as const;
+      const { user } = render(
+        <ChartDataProvider
+          width={400}
+          height={400}
+          margin={0}
+          series={series}
+          xAxis={[{ id: 'x', scaleType: 'band', data: [1, 2], position: 'none' }]}
```

Comment:
Such a modification would break the tooltip.

```suggestion
          xAxis={[{ id: 'x', scaleType: 'band', data: [2, 1], position: 'none' }]}
```